src/kvstore/raftex/RaftPart.cpp

```diff
@@ -763,6 +767,7 @@ void RaftPart::replicateLogs(folly::EventBase* eb,
 
     VLOG(2) << idStr_ << "About to replicate logs to all peer hosts";
 
+    lastMsgSentDur_.reset();
```

Comment:
Why move it here?  It is not the meaning of lastMsgSentDur_

Reply:
I've renamed the original `lastMsgSentDur_` to `lastMsgAcceptedDur_`, `lastMsgSentDur_` is used to record how long the net io time

Reply:
i think what you want is lastMsgSendingMs_ and lastMsgSentMs_

Reply:
Yeah, so I use `lastMsgAcceptedCostMs_` to store difference between lastMsgSendingMs_ and lastMsgSentMs_

Reply:
That's ok